examples/integration/test/fixtures/bwb.xcodeproj/xcshareddata/xcschemes/iOSAppObjCUnitTestSuite.xcscheme

```diff
@@ -14,9 +14,9 @@
                <EnvironmentBuildable>
                   <BuildableReference
                      BuildableIdentifier = "primary"
-                     BlueprintIdentifier = "288B82F8F3A8CB437DAEF64E"
+                     BlueprintIdentifier = "8EE630791B4B7821701DA6DE"
                      BuildableName = "iOSAppObjCUnitTestSuite.xctest"
-                     BlueprintName = "iOSAppObjCUnitTestsSuite"
+                     BlueprintName = "iOSAppObjCUnitTestSuite"
```

Comment:
Example 3: one has a `Tests` and one has `Test`, and it's better now that it's the bundle name version.